Batching/batching.md

```diff
@@ -303,9 +308,10 @@ Other areas highly likely to be bottlenecks:
 * Try steps to increase batching given above
 * Try switching 'single_rect_fallback' to on
 * The single rect fallback method is the default used without batching, and it is approximately twice as fast, however it can result in flicker on some hardware, so its use is discouraged
+* After trying the above, if your scene is still performaing worse, you may be fill-rate limited. In which case, consider turning off batching.
```

Comment:
Aside from the lights (which have scissoring), are there any situations where fill rate would be higher with batching that without? I can't immediately think of any so have left the fill rate limitation bit out for now.

Reply:
Not that I can think of. But I'm thinking back to my test scene that had a slowdown. It was using all large atlas textures and a ton of sprites. I attributed the slowdown to the bottleneck that occured as a result of using such a large texture so many times. To me, that seems related but not exactly a fill rate limitation. And for simplicity's sale may be best called a fill rate limitation.